src/App.tsx

```diff
@@ -1,61 +1,124 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { UserMas } from './interfaces/UserMas';
+import { TodosMas } from './interfaces/TodosMas';
+import { TodoList } from './components/TodoList';
+
+function getTodos(todoUserId: number): UserMas | undefined {
+  return usersFromServer.find(user => user.id === todoUserId);
+}
+
+function getNewPostId(posts: TodosMas[]) {
+  const maxId = Math.max(...posts.map(post => post.id));
+
+  return maxId + 1;
+}
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [isTitleError, setIsTitleError] = useState(false);
+
+  const [isUserId, setIsUserId] = useState(0);
```

Comment:
```suggestion
  const [userId, setUserId] = useState(0);
```
It's not boolean so `is` looks confusing